ci/cpu/build.sh

```diff
@@ -34,10 +35,18 @@ env
 logger "Activate conda env..."
 source activate gdf
 
+logger "Install Openjdk"
+conda install -c anaconda openjdk
```

Comment:
The anaconda channel should be part of `defaults` which is in the gpuci container `.condarc` so it doesn't need to be specified here. Would also suggest pushing this into https://github.com/rapidsai/gpuci-build-environment instead of doing it here.
```suggestion
conda install  openjdk
```